test-support/helpers/a11y/helpers/actions.js

```diff
@@ -14,7 +14,8 @@ const FOCUS_SELECTORS = [
   'a[href]',
   'area[href]',
   'iframe',
-  '[tabindex]:not([tabindex="-1"])'
+  '[tabindex]:not([tabindex="-1"])',
+  'form'
```

Comment:
@trentmwillis adding `'form'` to `FOCUS_SELECTORS` might not be the best approach. 

Since the functions declared in this file are part of the package's public API, I'm not sure of a better way.